csrc/scheduler/mma_utils.h

```diff
@@ -370,6 +374,25 @@ int64_t computeExpectedSharedMemoryUsage(
 //!  Z = complex<double>
 char dtypeToChar(const DataType& dtype);
 
+//! Get a total ordering of dimensions for known tensors. All dims of a
+//! particular DimRole are adjacent in the output. We then set the order as
+//! follows:
+//! 1. Batch dimensions go first
+//! 2. K dimensions are innermost
+//! 3. M or N can be innermost, depending on the first output's allocation
```

Comment:
> Oh, N don't have to be innermost?! Are we already supporting matmul-transpose fusion? In that case, we need to flip A and B based on the math $(AB)^T = B^T A^T$. Could you add a test for that if you are supporting it (let's do this in a separate PR).

That's the intention but i havent done the swap of the MmaOp inputs yet

Reply:
I could assert that N is innermost for now, or perhaps I could put a check in `isMatmulFusionDefinitionSupported` until this is supported. Asserting is probably fine though since the most common paths will all have N innermost. Since we don't support transposes in the epilogue I think we'd need to manually set the allocation domain on the output to specify a column-major output.

Reply:
Added assertion that N is innermost. Will revisit in another PR.